# DNA-driven condensation assembles the meiotic DNA break machinery

The accurate segregation of chromosomes during meiosis—which is critical for genome stability across sexual cycles—relies on homologous recombination initiated by DNA double-strand breaks (DSBs) made by the Spo11 protein1,2. The formation of DSBs is regulated and tied to the elaboration of large-scale chromosome structures3,4,5, but the protein assemblies that execute and control DNA breakage are poorly understood. Here we address this through the molecular characterization of Saccharomyces cerevisiae RMM (Rec114, Mei4 and Mer2) proteins—essential, conserved components of the DSB machinery2. Each subcomplex of Rec114–Mei4 (a 2:1 heterotrimer) or Mer2 (a coiled-coil-containing homotetramer) is monodispersed in solution, but they independently condense with DNA into reversible nucleoprotein clusters that share properties with phase-separated systems. Multivalent interactions drive this condensation. Mutations that weaken protein–DNA interactions strongly disrupt both condensate formation and DSBs in vivo, and thus these processes are highly correlated. In vitro, condensates fuse into mixed RMM clusters that further recruit Spo11 complexes. Our data show how the DSB machinery self-assembles on chromosome axes to create centres of DSB activity. We propose that multilayered control of Spo11 arises from the recruitment of regulatory components and modulation of the biophysical properties of the condensates.
